app/AppKernel.php

```diff
@@ -66,7 +66,7 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         if ($this->getEnvironment() === 'dev') {
             $loader->load(function (ContainerBuilder $container) {
                 // Check if webpack dev server is up before using it
-                @file_get_contents($container->getParameter('webpack_dev_server_base_url'));
+                @file_get_contents($container->resolveEnvPlaceholders($container->getParameter('webpack_dev_server_base_url'), true));
```

Comment:
```suggestion
                @file_get_contents($container->getParameter('webpack_dev_server_base_url'));
```

no need to resolve because this is a URL, because we already convert the env variable to a parameter:

```yaml
webpack_dev_server_base_url: '%env(WEBPACK_DEV_SERVER_BASE_URL)%'
```

and this is a URL, so I don't need we'll have to resolve anything 🤔